src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
```

Comment:
The tests failed. Try to avoid test failures by using a different way to write static and property values for objects in the code